client/test/Test/Pos/Client/Txp/Mode.hs

```diff
@@ -87,5 +84,5 @@ instance HasTxpConfigurations => MonadAddresses TxpTestProperty where
     getNewAddress = lift . getNewAddress
     getFakeChangeAddress = lift getFakeChangeAddress
 
-instance HasTxpConfigurations => Testable (TxpTestProperty a) where
-    property = monadic (ioProperty . flip runReaderT genesisBlockVersionData)
+instance (HasTxpConfigurations, Testable a) => Testable (TxpTestProperty a) where
```

Comment:
It's strange that `Testable a` somehow became needed here.

Reply:
`QuickCheck` API changed. In `develop` branch `QuickCheck-2.9.2` is used where [`monadic`](https://hackage.haskell.org/package/QuickCheck-2.9.2/docs/Test-QuickCheck-Monadic.html#v:monadic) has type:

```haskell
monadic :: Monad m => (m Property -> Property) -> PropertyM m a -> Property
```

Now in LTS-10.4 there's `QuickCheck-2.10.1` where:

```haskell
monadic :: (Testable a, Monad m) => (m Property -> Property) -> PropertyM m a -> Property
```